sdk/nodejs/policy/policy.ts

```diff
@@ -102,8 +102,67 @@ export interface Policy {
      * proper permissions.
      */
     enforcementLevel?: EnforcementLevel;
+
+    /**
+     * Configuration schema.
+     */
+    config?: PolicyConfigSchema;
```

Comment:
I keep going back and forth between calling this `config` vs. `schema` vs. `configSchema`. Any preferences? The latter is the most clear (though, slightly more verbose). [ESLint calls its equivalent `schema`](https://eslint.org/docs/developer-guide/working-with-rules#options-schemas) whereas [TSLint calls its equivalent `options`](https://github.com/palantir/tslint/blob/5ba66591baee97d0514c27bb106b1e4b62c7853e/src/rules/noAnyRule.ts#L47-L52).

Reply:
I prefer configSchema for the schema.

Reply:
Changed to `configSchema`.